src/main/java/org/dswarm/graph/index/MapDBUtils.java

```diff
@@ -83,6 +91,26 @@
 		return db.getAtomicLong(indexName);
 	}
 
+	public static Atomic.Long createOrGetLongIndex(final DB db, final String indexName, final long initValue) {
+
+		final Atomic.Long longIndex;
+
+		if (db.getCatalog().containsKey(indexName)) {
+
+			final Atomic.Long atomicLong = db.getAtomicLong(indexName);
+			atomicLong.getAndSet(initValue);
```

Comment:
Dunno about this `Atomic.Long`, but a regular `AtomicLong` has just `set` to convey the semantics of not being interested in the current value. Nothing dramatic I would change now, maybe just something to keep in mind for future uses :smiley:.